### ISSUE TITLE
Release please !

### ISSUE THREAD

**Issue by jmvanel** [shellac/java-rdfa#42]
That would be a nice 2016 new year present , after 3 years and a half since last release.

Especially needed for the Jena 3.X upgrade;
besides it's it's not convenient to build java-rdfa in Maven and build my app my SBT.

**Comment by doriantaylor** [comment]
agreed; it also seems like jena 3 has changed how location mapping works, as the `location-mapping.n3` needs massaging for the tests to pass.

**Comment by wglas85** [comment]
We have finally released java-rdfa-1.0.0-BETA1, which is now being developed in https://github.com/iteggmbh/java-rdfa
This is done in accordance with the original developer Damian Steer of the University of Oxford.
This release incorporates all jena-3 changes and moved the build to jena-3.16.0 and java-8
